packages/core/test/integration/model/findAll.test.js

```diff
@@ -1674,6 +1674,24 @@ The following associations are defined on "Worker": "ToDos"`);
       expect(info.rows[0].dataValues).to.not.have.property('username');
       expect(info.rows[1].dataValues).to.not.have.property('username');
     });
+
+    it('handles grouped rows', async function () {
+      const info = await this.User.findAndCountAll({
+        attributes: [
+          [Sequelize.fn('sum', Sequelize.col('intVal')), 'sum'],
+          Sequelize.col('intVal'),
+        ],
+        group: [Sequelize.col('intVal')],
+        countGroupedRows: true,
+        raw: true,
+      });
+      expect(info.count).to.equal(1);
+      expect(Array.isArray(info.rows)).to.be.ok;
+      const row = info.rows[0];
+      // MySQL returns sum as string, other dialects return number
+      row.sum = Number(row.sum);
```

Comment:
```suggestion
      if (dialectName === 'mysql) {
         row.sum = Number(row.sum)
      }
```
This conveys the same information as the comment, with the added benefit of not changing `row.sum` for the other dialects